### DISCUSSION TITLE
Plugin Not Showing in Streamdeck as of June 1st 2021

### DISCUSSION THREAD

**Discussion by JL-40** [fredemmott/StreamDeck-Discord#78] [Q&A] [answered]
For some reason, this plugin stopped working on my Streamdeck and doesn't even show up in the Streamdeck application. I've checked if my Streamdeck application is out of date and it is. I've also tried restarting the application and my computer, but nothing has changed. Is there something I'm missing? I'm on the most recent version of Windows 10 Pro (as of June 1st, 2021).

**Comment by MRT440** [comment]
I also have the same problem. Recently, the plugin is no longer recognised on my stream deck. And it is impossible to reinstall it because it tells me that it is already installed. But I can't find it in my plugins. Do you have a solution ?

**Comment by Nocker777** [comment]
I have the same issue.

**Comment by mattytwoface** [comment]
Me too.. any fix?

**Comment by fredemmott** [comment, marked as answer]
Try:

1. fully closing the streamdeck software by right clicking on it's icon and selecting 'close'/quit etc, not just closing the window
2. delete the plugin from `%appdata%\Elgato\StreamDeck\Plugins\` (windows) or `~/Library/Application Support/com.elgato.StreamDeck/Plugins/` (macos)
3. start the streamdeck software
4. re-install and re-configure the plugin
5. if using a pre-release, restart the streamdeck software again

Also check your antivirus/antimalware logs, especially if that doesn't fix this problem.

**Reply by Nocker777** [reply]
This worked for me - thank you!

**Reply by fredemmott** [reply]
Great to hear - this does mean that *something* external (maybe the streamdeck software itself, maybe windows defender/antivirus/...) was modifying/interferring with the plugin on disk though.

**Reply by JL-40** [reply]
Yup, this fixed it for me, thanks

**Reply by fredemmott** [reply]
If anyone else has this problem, or it reoccurs, I would greatly appreciate it if you could create an archive/zip/7z of the plugin folder before deleting it and send me a copy; I haven’t had this on any of machines, and my plugins never modify their installation directory, so it’s basically impossible to debug without this.

**Reply by mattytwoface** [reply]
Thank's a lot! Now it's working

**Reply by shawnhank** [reply]
Hi, I'm running StreamDeck on the macOS Monterrey Beta (10.12) and the Audio Mute plugin isn't working. I get that my mileage will vary since I'm running the beta. Just wanted to share that it's not working as expected.  

Tested this by using the Sound Preferences Input metering bar as well as recording my AirPods, built-in microphone, and an external mic. All resulted in the same outcome - Mute/Unmute/Toggle didn't work. 

I did go through the above steps to try and rectify the issue (Quit Stream Deck, Delete Plugins from the plugins folder, etc.) and it still ended up not working.  Will be rebooting after posting this to see if that fixes it. 

Interestingly, the Audio Switcher plugin is working as expected. So glad Apple didn't mess that one up for you. 😄 

[Here's a link to the plugins zip folder](https://1drv.ms/u/s!Ak2y55NEfts4kr8NBowSxb2ZU1ibqA?e=gndt0C) as requested using OneDrive. The folder is 62MB zipped and GH seems to not support that file size. 

Let me know if you have troubles accessing it.

**Reply by tatertacoma** [reply]
I am still having problems... I reinstalled both the Stream Deck application and all the plugins and it just doesn't work... Literally nothing except the bog-standard "Plugins" that come with the stream deck... My deck is completely broken... (Also, I never use Github but this is really annoying me...) Basically, anything that is an external plugin is broken, even the Phillips Hue (but that's always been broken)...

**Reply by fredemmott** [reply]
@tatertacoma sorry, you probably need to reach out to Elgato support; if no plugins are working, it's unlikely to be fixable by this plugin.
